orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -630,7 +625,7 @@ class TaskController {
 
   private List<Execution> filterPipelinesByHistoryCutoff(List<Execution> pipelines, int limit) {
     // TODO-AJ The eventual goal is to return `allPipelines` without the need to group + filter below (WIP)
-    def cutoffTime = (new Date(clock.millis()) - daysOfExecutionHistory).time
+    def cutoffTime = clock.instant().minus(daysOfExecutionHistory, DAYS).toEpochMilli()
```

Comment:
Another instance of using `Instant` rather than pulling in the optional Groovy jar for `Date` operators.